framework/test/BundleTest.cpp

```diff
@@ -287,6 +299,11 @@ void frame037a()
 
   US_TEST_CONDITION(buExec.GetBundleContext().GetDataFile("").substr(0, baseStoragePath.size()) == baseStoragePath, "Test for valid data path")
   US_TEST_CONDITION(buExec.GetBundleContext().GetDataFile("bla").substr(0, baseStoragePath.size()) == baseStoragePath, "Test for valid data file path")
+
+  US_TEST_CONDITION(buExec.GetBundleContext().GetProperty(Constants::FRAMEWORK_UUID).Empty() == false, "Test for non-empty framework uuid property")
+  auto props = buExec.GetBundleContext().GetProperties();
+  US_TEST_CONDITION_REQUIRED(props.empty() == false, "Test for non-empty bundle props")
+  US_TEST_CONDITION(props.count(Constants::FRAMEWORK_VERSION) > 0, "Test for existing framework version prop")
```

Comment:
Will we ever have more than one ```FRAMEWORK_VERSION``` property? If not, I'd suggest making this check for the existence of only one ```FRAMEWORK_VERSION``` property.